src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -184,7 +186,7 @@ def setup(self):
             name="token_type_embeddings",
             dtype=self.dtype,
         )
-        self.layer_norm = FlaxBertLayerNorm(hidden_size=self.config.hidden_size, name="layer_norm", dtype=self.dtype)
+        self.LayerNorm = FlaxBertLayerNorm(hidden_size=self.config.hidden_size, name="LayerNorm", dtype=self.dtype)
```

Comment:
Ugh, not happy to get that hideous name back :-( but I understand we have to match the PyTorch names!

Reply:
BTW the `name=` argument here is unncessary (and does nothing)